devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy

```diff
@@ -415,10 +419,13 @@ def configure() {
 		*/
 		configCmds += zigbee.writeAttribute(0xFC02, 0x0000, 0x20, 0x01, [mfgCode: manufacturerCode])
 		// passed as little-endian as a bug-workaround
-		configCmds += zigbee.writeAttribute(0xFC02, 0x0002, 0x21, "7602", [mfgCode: manufacturerCode])
-	} else if (device.getDataValue("manufacturer") == "Samjin") {
+		int sensitivity = (0x0276 * 100 / ((motionSensitivity as Integer) ?: 100)) as int // Motion sensitivity is inverse
+		configCmds += zigbee.writeAttribute(0xFC02, 0x0002, 0x21, DataType.pack(sensitivity, DataType.UINT16, true), [mfgCode: manufacturerCode])
+	} else if (device.getDataValue("manufacturer") == "Samjin") { // Current version
 		log.debug "Refreshing Values for manufacturer: Samjin "
-		configCmds += zigbee.writeAttribute(0xFC02, 0x0000, 0x20, 0x14, [mfgCode: manufacturerCode])
+		def y = (180 / -99 * (((motionSensitivity as Integer) ?: 100) - 100) as int) // Motion sensitivity is inverse
```

Comment:
This math is a little hard to follow. Since there are only five possible inputs what do you think about just creating a map to the desired value? I think that will be easier to understand.

Reply:
It's linear inverse transformation, the equation was simplified. I can extract it into a separate function.
The reason I thought about using the transformation function instead of just values is so that the sensitivity can be mapped into a simple range from 1% to 150%. (100% being the current DTH value). This way it allows for an easy scaling to sensitivity and the function can extended to a free input range (instead of an enum) or a sliding scale UI if support in future. 

Either way it fine, let me know and I can change it accordingly.

Reply:
Mostly agree with Tom here. Since the input values are currently constrained it would make sense to just have a map.

But if you do want to keep this in for future changes, I at least think you should break it out into a separate function where you explain the function a bit more in the comments.